services/QuillLMS/app/models/diagnostic_question_optimal_concept.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: diagnostic_question_optimal_concepts
+#
+#  id           :bigint           not null, primary key
+#  question_uid :string           not null
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  concept_id   :integer          not null
+#
+# Indexes
+#
+#  unique_diagnostic_question_optimal_concepts_uid_id  (question_uid,concept_id) UNIQUE
+#
+class DiagnosticQuestionOptimalConcept < ApplicationRecord
```

Comment:
Add a 'dependent: delete` directive on Question, or some other directive, to avoid this new table triggering a `ActiveRecord::InvalidForeignKey` error when a Question is deleted (it's an edge case, but might as well anticipate it now).

Reply:
Good call!  I'll update this for both `Question` and `Concept`.